sdks/go/pkg/beam/core/runtime/graphx/xlang.go

```diff
@@ -16,11 +16,222 @@
 package graphx
 
 import (
+	"fmt"
+
 	"github.com/apache/beam/sdks/go/pkg/beam/core/graph"
 	"github.com/apache/beam/sdks/go/pkg/beam/internal/errors"
 	pipepb "github.com/apache/beam/sdks/go/pkg/beam/model/pipeline_v1"
 )
 
+// MergeExpandedWithPipeline adds expanded components of all ExternalTransforms to the existing pipeline
+func MergeExpandedWithPipeline(edges []*graph.MultiEdge, p *pipepb.Pipeline) {
```

Comment:
Please un-export all functions if they aren't required outside of the graphx package. Naturally, keep any existing doc comments. No reason to remove them just because the functions aren't exported.

Reply:
Done.